Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getTypeMap.txt

MongoDB\\Collection::getTypeMap()
=================================

.. meta::

>    :description: Discover how to retrieve the type map for a MongoDB collection using the `getTypeMap()` method in PHP.

Reviewer: ```suggestion
   :description: Retrieve the type map for a collection with the MongoDB PHP Library.
```
